squarelet/organizations/fe_api/serializers.py

```diff
@@ -0,0 +1,129 @@
+# Django
+from django.db.models.expressions import F
+
+# Third Party
+import stripe
+from rest_framework import serializers, status
+from rest_framework.exceptions import APIException
+
+# Squarelet
+from squarelet.organizations.models import Invitation, Membership, Organization
+
+
+class OrganizationSerializer(serializers.ModelSerializer):
+    uuid = serializers.UUIDField(required=False)
+    # remove plan once all clients are updated to handle entitlements
```

Comment:
I think you can remove this comment - if we want the plan name to display, we can just leave it in this API